py/picca/pk1d/postproc_pk1d.py

```diff
@@ -205,229 +200,791 @@ def compute_mean_pk1d(p1d_table,
     One row per z bin; column values z_min/max, k_min/max, N_chunks
     """
```

Comment:
docstring needs fixing

Reply:
not sure what you mean here, fixed the docstrings to be numpy-docstring compatible according to pydocstyle (maybe this should be run at linting time as well, to have properly formatted docstrings), the line cited is just regarding the format the table needs, which I think is described reasonably well...

Reply:
I mean that there are several keyword arguments that are not described on the docstring. Basically all these
```
nomedians=False,
    velunits=False,
    output_snrfit=None,
    compute_covariance=False,
    compute_bootstrap=False,
    number_bootstrap=50,
    number_worker=8
```

Reply:
Ah, I'd say we can merge with an incomplete docstring given that we want to update after anyway. And most of those variable names are self-explanatory anyway... But if you insist, I guess one could add those

Reply:
I've completed the docstring  on covariance options (the rest is documented in either of the functions).